sdk/keyvault/azure-keyvault-secrets/samples/README.md

```diff
@@ -4,8 +4,8 @@ languages:
   - python
 products:
   - azure
+  - azure-key-vault
 ---
-
```

Comment:
Every other file here leaves this blank line above the first heading

Reply:
@bryevdv  thanks! missed it, fixed now